resource.xml

```diff
@@ -0,0 +1,28 @@
+<assembly>
+    <id>resources</id>
+    <formats>
+        <format>zip</format>
+    </formats>
+    <includeBaseDirectory>false</includeBaseDirectory>
+    <fileSets>
+        <fileSet>
+            <directory></directory>
+            <outputDirectory></outputDirectory>
+
+            <excludes>
+                <exclude>pom.xml</exclude>
+                <exclude>*.iml</exclude>
+                <exclude>**/src/**</exclude>
+                <exclude>**/target/**</exclude>
+                <exclude>resource.xml</exclude>
+                <exclude>**/.idea/**</exclude>
+            </excludes>
+
+            <includes>
+                <include>lib</include>
```

Comment:
Fixed the lib directory include in commit 7b4454fb81d04eaf19feeb68dc5f7a057e2e6035